swagger/v1/component_schemas/ListFilter.yml

```diff
@@ -5,3 +5,7 @@ properties:
     description: "Return only records that have been updated since this date and time (ISO 8601 date format)"
     type: :string
     example: "2021-05-13T11:21:55Z"
+  participant_id:
+    description: "The unique id of the participant"
+    type: :string
+    example: "ab3a7848-1208-7679-942a-b4a70eed400a"
```

Comment:
Hmm, perhaps at some point we should split it, cause maybe not all of our filters will make sense for all the endpoints, but it's probably ok for now.